h/views/api_groups.py

```diff
@@ -41,6 +42,10 @@ def groups(request):
             description='Create a new group')
 def create(request):
     """Create a group from the POST payload."""
+    # @TODO Temporary: Remove this check once private groups supported at other authorities
+    if request.authority != request.user.authority:
+        raise HTTPBadRequest(_('Group creation currently only supported for default authority'))
```

Comment:
I think I'm a little bit confused about what's going on here. So if an eLife user, for example, called this API, `request.authority` would still be `"hypothes.is"` (the default authority set by the `AUTHORITY` env var) but `request.user.authority` would be `"elifesciences.org"`?